code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -58,9 +58,11 @@
 /obj/item/modular_computer/tablet/add_context(atom/source, list/context, obj/item/held_item, mob/user)
 	. = ..()
 
-	context[SCREENTIP_CONTEXT_CTRL_LMB] = "Remove pen"
+	if(istype(inserted_item, /obj/item/pen))
+		context[SCREENTIP_CONTEXT_CTRL_LMB] = "Remove pen"
```

Comment:
If(!qdeleted(inserted_item))
"Remove [inserted_item]"

Inserted item handles more than just pens, nut they're always removed with altclick.

Reply:
Why are we using `qdeleted` for this cause that seems like overkill?  We don't actually use the `inserted_item` for anything involving our SCREENTIPS other than to check if it exists so `if(inserted_item)` should be sufficient.

Reply:
Changed pen to be inserted_item in 8009b43e2c70b979f4802a39fd80f409bb85d656